src/libs/actions/IOU.js

```diff
@@ -1014,7 +1016,6 @@ function getSendMoneyParams(report, amount, currency, comment, paymentMethodType
  * @returns {Object}
  */
 function getPayMoneyRequestParams(chatReport, iouReport, recipient, paymentMethodType) {
-    const reportPreviewAction = ReportActionsUtils.getReportPreviewAction(chatReport.reportID, iouReport.reportID);
```

Comment:
Why are we removing the optimistic action?

Reply:
This is what was in the end causing the bugs with payment. if you sign in and navigate dirrectly to the IOU report and click pay, we dont have the report preview action as it was not treated as the parent report action -> hence the auth change. So the app crashed here/ you could not pay.